core/server/worker/src/main/java/alluxio/worker/block/BlockLockManager.java

```diff
@@ -111,7 +111,18 @@ public long lockBlock(long sessionId, long blockId, BlockLockType blockLockType)
       }
       lock = blockLock.writeLock();
     }
-    lock.lock();
+    try {
+      if (!lock.tryLock(1L, TimeUnit.MINUTES)) {
+        LOG.warn("tryLock {} block {} for session {} failed after 1 min, "
```

Comment:
no need to log session since we didn't track session correctly after Alluxio 1.5

Reply:
I think we can distinguish where the lock requester come from, If it come from `AsyncBlockRemover`, the sessionId could be `MASTER_COMMAND_SESSION_ID=-4`

Reply:
Make sense!